plugin/hover.py

```diff
@@ -99,11 +99,11 @@ def run_async() -> None:
                 self.request_symbol_hover_async(listener, hover_point)
             self._diagnostics_by_config, covering = listener.diagnostics_touching_point_async(hover_point)
             if self._diagnostics_by_config:
-                if not only_diagnostics:
-                    actions_manager.request_with_diagnostics_async(
-                        self.view, covering, self._diagnostics_by_config,
-                        functools.partial(self.handle_code_actions, listener, hover_point))
                 self.show_hover(listener, hover_point, only_diagnostics)
+            if not only_diagnostics:
+                actions_manager.request_for_region_async(
+                    self.view, covering, self._diagnostics_by_config,
+                    functools.partial(self.handle_code_actions, listener, hover_point))
```

Comment:
Is this code shuffle intentional?

Reply:
> Is this code shuffle intentional?

Whether the diagnosis exists or not, the `code action` needs to be executed. They have to be independent, Any questions

Reply:
Right, it does seem like quite a bit more expensive but correct approach.